.changeset/fifty-weeks-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Fix build issues for `node:*`, for supported runtime APIs
```

Comment:
It's a minor but we write as it was a patch 😅

Reply:
Still waiting for docs, so can push an update. 🚀